releasing/plugin-commands-deploy/test/deploy.test.ts

```diff
@@ -46,6 +59,10 @@ test('deploy', async () => {
     fs.writeFileSync(`${name}/index.js`, '', 'utf8')
   })
 
+  await writeYamlFile('pnpm-workspace.yaml', { packages: ['*'] })
```

Comment:
In order to `createExportableManifest` we need metadata only available after an install.

Presently, `deploy` does not actually require an install previously.

This _could_ be considered a breaking change. Although I would assume most users already have the relevant packages installed, and expected this behavior. This is inline with how commands like `publish` work.

`deploy` could be considered a "local" `publish`.